lib/fs.js

```diff
@@ -130,7 +149,7 @@ function isFd(path) {
   return (path >>> 0) === path;
 }
 
-// Static method to set the stats properties on a Stats object.
+// Constructor for file stats
 function Stats(
```

Comment:
btw, it's worth noting that I had tried to convert this to a ES6 Class just to see what would happen and it yielded a 50% perf *reduction*.